www/js/lib/uiUtil.js

```diff
@@ -227,12 +228,83 @@ define([], function() {
             '/': '&#x2F;',
             '`': '&#x60;',
             '=': '&#x3D;'
-          };
-          string = String(string).replace(/[&<>"'`=/]/g, function (s) {
-            return escapechars[s];});
-          return string;
+        };
+        string = String(string).replace(/[&<>"'`=/]/g, function (s) {
+            return escapechars[s];
+        });
+        return string;
+    }
 
+    /**
+     * Removes the animation effect between various sections
+     */
+    function removeAnimationClasses() {
+        $('#about').removeClass('slideIn_L').removeClass('slideOut_R');
+        $('#configuration').removeClass('slideIn_L').removeClass('slideIn_R').removeClass('slideOut_L').removeClass('slideOut_R');
+        $('#articleContent').removeClass('slideIn_R').removeClass('slideOut_L');
     }
+    
+    /**
+     * Adds the slide animation between deifferent sections
+     * 
+     * @param {String} name_of_section It takes the name of the section to which the animation is to be added
+     * 
+     */
+    function addAnimation(section) {
```

Comment:
I would suggest to rename this fonction to something like animateToSection, because it does not simply add classes

Reply:
Yes,I have done the renaming....that sounds more relative

Reply:
@mossroy @vasusharma7 It's not very important, but `animateToSection` is odd English. Either: "animateSection", or "applyAnimationToSection", or "applyCSSAnimationToSection" would be more natural.

Reply:
Yes, applyAnimationToSection seems more appropirate.
I will change this in my next commit along with other changes.
I am waiting for @mossroy 's comments on the (https://github.com/kiwix/kiwix-js/pull/579#discussion_r365526418) conversation and will submit only a single commit.